style.css

```diff
@@ -1,3 +1,69 @@
-#root {
-  color: red;
+html {
+  font-size: 16px;
+}
+
+.allEpisodes {
```

Comment:
Nice use of flexbox for you episode content. Would be good to try and apply that to your navbar too.